redis/redis_test.go

```diff
@@ -146,7 +146,7 @@ func Test_Redis_Initalize_WithURL(t *testing.T) {
 }
 
 func Test_Redis_Initalize_WithURL_TLS(t *testing.T) {
-	cer, err := tls.LoadX509KeyPair("./tests/tls/client.crt", "./tests/tls/client.key")
+	cer, err := tls.LoadX509KeyPair("/home/runner/work/storage/storage/tls/client.crt", "/home/runner/work/storage/storage/tls/client.key")
```

Comment:
Why don't you use relative path here

Reply:
It's easier to read rather than "../../../path/to/cerr"

Reply:
> It's easier to read rather than "../../../path/to/cerr"

But it won't work on local machines

Reply:
I have never been able to run these tests locally. I can change it if needed, the certs are no longer just for Redis, they can be used by other drivers